### ISSUE TITLE
Buggy duplicate pushstate and back button

### ISSUE THREAD

**Issue by alanontheweb** [terwanerik/AjaxPageParser#3]
I really like this plugin, and want to continue using it but have spotted 2 significant bugs. I was hoping you could take a look and perhaps offer a fix when possible. It's hard to use in production while these issues are pending.

1)When visiting the demo page - http://terwanerik.github.io/AjaxPageParser/index.html

Refreshing the page several times keeps adding a pushState to the browser's history, making the back button not actually link to the previous page. It shouldn't be adding empty history states like that.

<img width="1000" alt="screen shot 2016-08-25 at 8 37 35 pm" src="https://cloud.githubusercontent.com/assets/302498/17993361/038bd97c-6b04-11e6-9e8f-b2826d3f6c63.png">

---

2) Further, when you open the demo in a new tab and navigate to the second page, and then hit "back" on the browser, the address bar gets updated, but the script does not replace the ajax content region.

<img width="686" alt="screen shot 2016-08-25 at 8 42 02 pm" src="https://cloud.githubusercontent.com/assets/302498/17993411/6cab56ee-6b04-11e6-9d83-468a80d982b9.png">

**Comment by terwanerik** [comment]
Yeah i've been planning a rewrite for this, in vanilla JS, for a while now. When/if i have some free time i'll start work on that, fixing this, and other things, that are wrong with this plugin.
